README.md

```diff
@@ -0,0 +1,48 @@
+# NYPL Header App
+The NYPL Header App can be used for projects across the NYPL organization to import the Header and Footer components.
+
+# Running the app locally
```

Comment:
Let's have one "h1" heading at the very top. So this and the other should be `##`. "Running the app locally with Docker" is a subheading so that can be `###`.